panels/background/cc-background-panel.c

```diff
@@ -473,16 +431,7 @@ reload_current_bg (CcBackgroundPanel *self)
   g_free (uri);
 
   /* initalise the current background information from settings */
-  uri = g_settings_get_string (priv->settings, WP_URI_KEY);
-
-  /* initialise from language if required */
-  if (g_strcmp0 (uri, EOS_DEFAULT_BG_URI) == 0)
-    {
-      g_free (uri);
-      uri = get_endless_default_bg_uri ();
-    }
-
-  /* fall through the former logic otherwise */
+  uri = g_settings_get_string (settings, WP_URI_KEY);
```

Comment:
Is the change from "priv->settings" to "settings" correct or an unintentional consequence of a conflict?